tensorflow/python/keras/layers/convolutional.py

```diff
@@ -199,21 +199,8 @@ def call(self, inputs):
           # nn.bias_add does not accept a 1D input tensor.
           bias = array_ops.reshape(self.bias, (1, self.filters, 1))
           outputs += bias
-        if self.rank == 2:
+        else:
           outputs = nn.bias_add(outputs, self.bias, data_format='NCHW')
-        if self.rank == 3:
-          # As of Mar 2017, direct addition is significantly slower than
```

Comment:
Has this changed since? Does `bias_add` support 5D inputs in NCDHW mode?

Reply:
Yes, `bias_add` has supported 5D input, see PR: https://github.com/tensorflow/tensorflow/pull/22127